nextcloudappstore/core/tests/test_app_ownership_transfer.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from nextcloudappstore.core.models import App, AppOwnershipTransfer
+
+
+class AppOwnershipTransferTest(TestCase):
+    def setUp(self):
+        self.user1 = User.objects.create_user(username='user1')
```

Comment:
AFAIK things run in setUp are not included in the transaction and you need to delete them again in the tearDown method. I'd probably move them to a separate method and call that one in each relevant block. Alternative is to delete them in a tearDown method